hun.json

```diff
@@ -0,0 +1,13 @@
+{
+    "loadButtonLabel": "Kattintson ide\na Panoráma\nBetöltéséhez",
+    "loadingLabel": "Betöltés...",
+	"bylineLabel": "%s által",    
+
+    "noPanoramaError": "Nincs megadott panorámakép.",
```

Comment:
Nincs megadva a panoráma kép.

Reply:
google translate agrees with this proposal

Reply:
I agree with that, though only with one half. The other changed word, 'a', translates to 'the' in English, with which it would mean "The panorama image was not specified", which IMO means something very slightly different. I would then suggest "Nincs megadva panoráma kép".